### ISSUE TITLE
Unable to retrieve child resources.  Details: ["Error:Unable to retrieve authentication token."]

### ISSUE THREAD

**Issue by aureole82** [microsoft/AzureStorageExplorer#3646]
Unable to retrieve child resources.

Details:
["Error:Unable to retrieve authentication token."]

![grafik](https://user-images.githubusercontent.com/4819891/95459582-25cd4d00-0974-11eb-80b7-651932b6004e.png)

**Comment by MRayermannMSFT** [comment]
@aureole82 can you go to Settings -> Services -> Disks, uncheck the checkbox and then restart? That should unblock you (assuming you don't need to work with disks).

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. If you are still having problems, please open a new issue. Thanks.
